src/target.c

```diff
@@ -355,22 +368,35 @@ stumpless_get_default_target( void ) {
 }
 
 int
-stumpless_get_option( const struct stumpless_target *target, int option ) {
+unchecked_get_option( const struct stumpless_target *target, int option ) {
   int options;
 
-  if( !target ) {
-    raise_argument_empty( L10N_NULL_ARG_ERROR_MESSAGE( "target" ) );
+  if ( !target ) {
```

Comment:
I believe this check is why your coverage has dropped a little - you can remove it. The `unchecked_` family of functions explicitly put the burden of checks on the caller, and in the usages of this function `target` has already been checked for NULL earlier.

I'll add some documentation on this naming convention and expectation as well as some others that up until this point have just existed in my head. These will go in `docs/style.md` shortly.